src/cli.ts

```diff
@@ -185,7 +185,8 @@ async function main(): Promise<void> {
                 tslint: "tslint -p tsconfig.json './src/**/*.ts'",
                 typecheck: 'tsc -p tsconfig.json',
                 build: `parcel build --out-file dist/${name}.js src/${name}.ts`,
-                serve: `parcel serve --no-hmr --out-file dist/${name}.js src/${name}.ts`,
+                symlinkPackage: '[ ! -f dist/package.json ] && (mkdir -p dist && pushd dist && ln -s ../package.json package.json && popd) || echo "dist/package.json already symlinked"',
```

Comment:
This won't work on Windows because of bash syntax, symlink command and symlinks not always being available in general.

Also, scripts in package.json are usually kebab-case not camelCase.

Reply:
Changed this to use a node script relying on [create-symlink](https://www.npmjs.com/package/create-symlink), which uses graceful-fs under the hood to handle this gracefully across platforms.

Also fixed case.